addons/binding/org.openhab.binding.fronius/src/main/java/org/openhab/binding/fronius/handler/FroniusHandler.java

```diff
@@ -273,13 +275,17 @@ private PowerFlowRealtimeResponse getPowerFlowRealtime(String ip) {
             if (!resultOk) {
                 logger.error("Error in fronius response: {}", errorMsg);
             }
-
+            connectionFailedPowerFlow = false;
         } catch (JsonSyntaxException e) {
             errorMsg = "Configuration is incorrect";
             logger.error("Error running fronius request: {}", errorMsg);
         } catch (IOException | IllegalStateException e) {
-            errorMsg = e.getMessage();
-            logger.error("Error running fronius request: {}", errorMsg);
+            if (!connectionFailedPowerFlow) {
+                logger.error("Error running fronius request: {}", e.getMessage());
+                connectionFailedPowerFlow = true;
+            } else {
+                logger.debug("Error running fronius request: {}", e.getMessage());
```

Comment:
What is the purpose of introducing the variable if both log statements look the same? Don't you want to adjust at least one of them?

Reply:
See the conversation above.
It's posible that the device goes offline at night and the ip isn't avaible anymore. 
I can not tell if it's a mistake or not, so because of the refresh loop i only log the first error as an error

Reply:
Alright, thanks for clarifying. if its the same error and the difference is that you only want to log it once in one case, that's fine for me.